.changeset/young-sheep-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@backstage/plugin-graphiql": patch
```

Comment:
Using single quotes here `'` should fix your prettier error.

You can also do `yarn run prettier --write <file>`.

Reply:
Thank you!